src/plugins/base64/README.md

```diff
@@ -99,7 +99,7 @@ Some file formats such as [YAML](http://yaml.org) already support Base64 encoded
 
 The diagram below shows how the Base64 conversion process works in conjunction with the YAML CPP plugin.
 
-![Tree](./base64/Base64.pdf)
+![Tree](./Base64.pdf)
```

Comment:
Please also make it lower-case, see #1615

Reply:
Actually, you can keep it that way. It seems like getting everything lower-case is impossible anyway. (e.g. README.md, CMakeLists.txt, ...)